pkg/packagekit/updates.jsx

```diff
@@ -948,7 +948,7 @@ class OsUpdates extends React.Component {
                           ? <div className="alert alert-warning">
                                 <span className="pficon pficon-warning-triangle-o"></span>
                                 <span>
-                                    <strong>{_("Cockpit itself will be updated.")}</strong>
+                                    <strong>{_("This web interface itself will be updated.")}</strong>
                                     &nbsp;
                                     {_("When you get disconnected, the updates will continue in the background. You can reconnect and resume watching the update progress.")}
```

Comment:
"Your browser will disconnect, but updates will continue in the background. You can reconnect in a few moments to see the progress."

Reply:
It would be nice to have a refresh that detects the state of the page and reloads when Cockpit is available again. (But this is completely out of scope for this PR.)

Reply:
I'd like to remove the duplication of the word "will" in this string, but it's tough to do so.

Reply:
Perhaps this:

"Your browser will disconnect, but this does not affect the update process. You can reconnect in a few moments to continue watching the progress."